var/spack/repos/builtin/packages/qt-svg/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+from spack.pkg.builtin.qt_base import QtBase, QtPackage
+
+
+class QtSvg(QtPackage):
+    """Scalable Vector Graphics (SVG) is an XML-based language for describing
+    two-dimensional vector graphics. Qt provides classes for rendering and
+    displaying SVG drawings in widgets and on other paint devices."""
+
```

Comment:
I realize there is a default `qt` homepage but is it worth adding a more specific one here (i.e., `https://doc.qt.io/qt-6/qtsvg-index.html`)?

Reply:
This probably makes sense, but not just for qt-svg: all the Qt packages except for qt-base and qt-declarative could do so. Here is a draft for changing that: #40850